### DISCUSSION TITLE
垃圾评论过滤功能

### DISCUSSION THREAD

**Discussion by finisky** [walinejs/waline#427] [Q&A] [answered]
最近刚切到Waline上，然后就被攻击了，见这里：https://finisky.github.io/zichao/

IP，nickname，Email都齐全。为避免这种攻击，可能需要加上验证码功能或关键词过滤了。。。

**Comment by Mister-Hope** [comment, marked as answer]
攻击者使用的是一言API，目前正在无差别攻击所有使用 Waline 的网站。关键词过滤似乎不是一个可行的好办法。

**Reply by finisky** [reply]
好奇他怎么找到使用waline的这些网站的

**Reply by Mister-Hope** [reply]
通过搜索引擎吧，vercel.app后缀就可以一搜一大把。另外用户群有用户反馈到百度搜索进入的次数大量增加。
